GamesProject/style.css

```diff
@@ -0,0 +1,47 @@
+
+a{
+    color: var(--color-1, #EF4934);
+    text-decoration-line: underline;
+}
+/* Для майбутньої верстки */
+.content{
+    width: 1175px;
+    height: 479px;
+    margin: 140px 52px 398px 53px;
+}
+
+p{
+color: var(--color-2, #221F1F);
+font-family: Roboto;
+font-size: 21px;
+font-style: normal;
+font-weight: 400;
+line-height: 32px;
+margin: 0;
+}
+
+.about{
+width: 575px;
+height: 479px;
+margin-left: 653px;
+display: inline-flex;
+flex-direction: column;
+align-items: flex-start; 
+gap: 24px;
+}
+
+.text-title{
+    color: var(--color-2, #221F1F);
+    width: 575px;
```

Comment:
It is bad approach to fix `width` and `height` simultaneously. Cause it can lead to broken layout, if a content-manager like to change the text of the header
<img width="683" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1a8beb7a-43a6-4337-b151-3147591a017b">